src/components/mastra/workspace/MastraFlowWorkspace.tsx

```
@@ -0,0 +1,511 @@
+import { NodeInspector } from '../inspector/NodeInspector';
+import { ExecutionDashboard } from '../dashboard/ExecutionDashboard';
+import { BuilderModal } from '../modals/BuilderModal';
+import { AgentBuilder } from '../builders/AgentBuilder';
+import { ToolBuilder } from '../builders/ToolBuilder';
+import { WorkflowBuilder } from '../builders/WorkflowBuilder';
+import { MemoryBuilder } from '../builders/MemoryBuilder';
+import { AgentConfig, ToolConfig, WorkflowConfig, MemoryConfig } from '@/lib/mastra/types/primitiveTypes';
+import { TemplateGallery } from '../templates/TemplateGallery';
+import { MastraTemplate } from '@/lib/mastra/templates/definitions';
+import { PatternBuilder, AtomicNodePattern } from '../patterns/PatternBuilder';
+import { PatternBuilder, AtomicNodePattern } from '../patterns/PatternBuilder';
```

Biome (2.1.2)

[error] Shouldn't redeclare 'PatternBuilder'. Consider to delete it or rename it.

'PatternBuilder' is defined here:


(lint/suspicious/noRedeclare)